api/src/models/Config.ts

```diff
@@ -219,6 +219,8 @@ class Config {
 
     get samlEntryPoint(): string {
         return (this.authenticationSettings["saml_entry_point"] as string) ?? "";
+    get licenses(): Record<string, string> {
```

Comment:
There appears to be a merge error here -- two functions have gotten incorrectly combined. You should just need to insert a close brace and newline above `get licenses()` to fix it.